interp/stdarg.ml

```diff
@@ -34,6 +34,9 @@ let wit_string : string uniform_genarg_type =
 let wit_pre_ident : string uniform_genarg_type =
   make0 "preident"
 
+let wit_scope_name : string uniform_genarg_type =
```

Comment:
This is a confusion between parsing and semantics. Do not add new generic arguments if there is no need for it.

Reply:
Does the following qualify as "needing it"?

![image](https://user-images.githubusercontent.com/1253341/212756678-662a94e0-367c-449e-a7a3-842b0fe52f74.png)

Reply:
This creates a new argument and thus a new runtime tag. Don't do that.